client/src/components/TodayScheduleTable/index.js

```diff
@@ -0,0 +1,367 @@
+          );
+        }
+        return text;
+      },
+    },
+    {
+      title: 'Appointment time',
+      dataIndex: 'appointmentTime',
+      width: '17%',
+      defaultSortOrder: 'ascend',
+      sorter: {
+        compare: (a, b) =>
+          moment(a.appointmentTime, 'hh:mm:ss') -
+          moment(b.appointmentTime, 'hh:mm:ss'),
+        multiple: 3,
+      },
+      render: (text, { key, appointmentTime }) => {
+        if (editingKey && editingKey === key) {
+          return (
+            <TimePicker
+              defaultValue={moment(appointmentTime, 'hh:mm:ss')}
+              onChange={onTimeChange}
+            />
+          );
+        }
+        return text;
+      },
+    },
+    {
+      title: 'operation',
+      dataIndex: 'operation',
+      render: (_, record) => {
+        const editable = isEditing(record);
+        return (
+          <div>
+            <Popconfirm
+              title="Sure to Delete?"
+              disabled={record.isDone}
+              onConfirm={() => deleteCell(record.key)}
+            >
+              <DeleteOutlined
+                style={{ color: 'red', fontSize: '21px', margin: '0 10px' }}
+                disabled={editingKey !== ''}
+              />
+            </Popconfirm>
+            {editable ? (
+              <span>
+                <CheckOutlined
+                  style={{ color: 'red', fontSize: '21px', margin: '0 10px' }}
+                  onClick={() => save(record.key)}
+                />
+
+                <Popconfirm title="Sure to check?" onConfirm={cancel}>
+                  <CloseOutlined
+                    style={{
+                      color: 'green',
+                      fontSize: '21px',
+                      margin: '0 10px',
+                    }}
+                  />
+                </Popconfirm>
+              </span>
+            ) : (
+              <EditOutlined
+                style={{ fontSize: '21px', margin: '0 10px' }}
+                onClick={() => (record.isDone ? null : edit(record))}
+              />
+            )}
+          </div>
+        );
+      },
+    },
+  ];
+  // const mergedColumns = columns.map((col) => {
+  //   if (!col.editable) {
+  //     return col;
+  //   }
+
+  //   return {
+  //     // ...col,
+  //     // onCell: (record) => ({
+  //     //   record,
+  //     //   inputType: col.dataIndex === 'age' ? 'number' : 'text',
+  //     //   dataIndex: col.dataIndex,
+  //     //   title: col.title,
+  //     //   editing: isEditing(record),
+  //     // }),
+  //   };
+  // });
+  return (
+    <Form form={form} component={false}>
+      {loading ? <h1>loading</h1> : null}
+      {error && <Alert message={error} type="error" showIcon />}
+      <Table
+        style={{
+          width: '60%',
+        }}
+        bordered
+        dataSource={srvData}
+        columns={columns}
+        rowClassName="editable-row"
+        pagination={{
+          onChange: cancel,
+        }}
+      />
+    </Form>
```

Comment:
This is what you should do when you import the `Loading` and `AlertMessage` common components

```suggestion
    <Form form={form} component={false}>
      {loading ? (
        <Loading />
      ) : error ? (
        <AlertMessage
          message="Error"
          type="error"
          description={error}
          showIcon
        />
      ) : (
        <Table
          style={{
            width: '60%',
          }}
          bordered
          dataSource={srvData}
          columns={columns}
          rowClassName="editable-row"
          pagination={{
            onChange: cancel,
          }}
        />
      )}
    </Form>
```

Reply:
done